include/tunnler/room_member.h

```diff
@@ -132,6 +132,12 @@ class RoomMember final {
     MacAddress mac_address; ///< The mac_address of this member.
 
     /**
+     * Extracts a chat entry from a received RakNet packet and adds it to the chat queue.
+     * @param packet The RakNet packet that was received.
+     */
+    void HandleChatPackets(const RakNet::Packet* packet);
```

Comment:
I was thinking of this, too. But changing it for wifiPackets isn't within the scope of this PR